index.js

```diff
@@ -66,13 +66,12 @@ const ERROR_MESSAGE_PARSING_FAILED = 'ps output parsing failed';
 const psFields = 'pid,ppid,uid,%cpu,%mem,comm,args';
 
 // TODO: Use named capture groups when targeting Node.js 10
-const psOutputRegex = /^[ \t]*(?<pid>\d+)[ \t]+(?<ppid>\d+)[ \t]+(?<uid>\d+)[ \t]+(?<cpu>\d+\.\d+)[ \t]+(?<memory>\d+\.\d+)[ \t]+/;
+const psOutputRegex = /^[\s]*(?<pid>\d+)[\s]+(?<ppid>\d+)[\s]+(?<uid>\d+)[\s]+(?<cpu>\d+\.\d+)[\s]+(?<memory>\d+\.\d+)[\s]+(?<comm>[\S]+(?:\s+<defunct>)?)[\s]+(?<args>.*)/;
```

Comment:
Wouldn't it fail if `comm` contains spaces(or other whitespace characters)?

Reply:
Yes, will fail. But: have you seen whitespace symbols in **comm** column? Specify environment if you seen, please.
On my linux environment can be one whitespace only in case with `<defunct>`

Reply:
run `node -e 'process.title = "ohnoes /bin/sh"; setTimeout(() => {}, 1<<30);'`

Reply:
Also
```
cp `which sleep` './help space'
./help\ space 10000
```

Reply:
Rewritten column separation approach.
For now, used **COMMAND** position in first line from `ps` output